src/blog/v8-release-85.md

```diff
@@ -0,0 +1,145 @@
+---
+title: 'V8 release v8.5'
+author: 'Zeynep Cankara, tracking some Maps'
+avatars:
+ - 'zeynep-cankara'
+date: 2020-07-21
+tags:
+ - release
+description: 'V8 release v8.5 features Promise.any, AggregateError, String.prototype.replaceAll, logical assignment operators, Liftoff shipped on all platforms, multi-value support shipped and support for JS BigInts'
```

Comment:
Let's reword to shorten this up a bit.

Reply:
Maybe at least

```suggestion
description: 'V8 release v8.5 features Promise.any, .replaceAll on strings, logical assignment operators, WebAssembly multi-value and BigInt support, and performance improvements'
```

Reply:
Could also use the `String#replaceAll` notation (we're doing it all over the place on v8.dev)

Reply:
Oh yeah right, that's probably even better.

Reply:
Updated.

Reply:
(this got accidentally reverted in a follow-up commit, marking as unresolved)